ViViaN/vivianInstall.sh

```diff
@@ -41,40 +45,67 @@ if [[ -z $scriptdir ]]; then
     scriptdir=/opt/vista
 fi
 
+if [[ -z $nonCache ]]; then
+    nonCache=false
+fi
+
 yum install -y httpd graphviz java-1.8.0-openjdk-devel php
 curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" && \
     python get-pip.py && \
     pip install xlrd reportlab
 # cp $scriptdir/ViViaN/viv.conf /etc/httpd/conf.d
-basedir=/opt/cachesys/$instance
+
+if ! $nonCache; then
```

Comment:
the double negative here is confusing. I'd flip this around since there is an else statement attached.